### PR TITLE
add SequenceMask op + tests

### PR DESCRIPTION
**Pull request by farizrahman4u**
## What changes were proposed in this pull request?

Add SequenceMask op

## How was this patch tested?

Unit tested in SameDiffTests

### CODE REVIEW

**Comment by farizrahman4u** [comment]
Note : Merge only after https://github.com/deeplearning4j/libnd4j/pull/849 is merged.

File: nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/SequenceMask.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    @Override
+    public void initFromTensorFlow(NodeDef nodeDef, SameDiff initWith, Map<String, AttrValue> attributesForNode, GraphDef graph) {
+        TFGraphMapper.getInstance().initFunctionFromProperties(nodeDef.getOp(), this, attributesForNode, nodeDef, graph);
+        addArguments();
+    }
+    @Override
+    public Map<String, Map<String, PropertyMapping>> mappingsForFunction() {
+        Map<String, Map<String, PropertyMapping>> ret = new HashMap<>();
+        Map<String, PropertyMapping> attrs = new LinkedHashMap<>();
+        if (maxLen >= 0) {
```
Comment (raver119) [inline nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/SequenceMask.java]: How this condition is going to happen during initialization from TF?

All ops during import are initialized with default constructor, so maxLen value will be always -1.

File: nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/SequenceMask.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    @Override
+    public String onnxName() {
+        throw new NoOpNameFoundException("No onnx opName found for " + opName());
+    }
+
+
+    @Override
+    public String[] tensorflowNames() {
```
Comment (raver119) [inline nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/SequenceMask.java]: You are implementing wrong method here, and with this method implemented only import wont work.

You should be implementing tensorflowName() instead.

Comment (raver119) [reply nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/SequenceMask.java]: P.s. if not overriden that method throws: https://github.com/deeplearning4j/nd4j/blob/930ad274fd2f8efaf3e73119cc9fb009b8f84571/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/DynamicCustomOp.java#L688-L690

But method tensorflowName() is called at line 64 in your op.

**Pull request merged**